### PR TITLE
[11.0][ADD] backport attachment unindex content to 11.0

### PR DESCRIPTION
**Pull request by randall-vx**

### CODE REVIEW

**Review by moylop260 — APPROVED** [review]
👍 
database_cleanup error unrelated

File: attachment_unindex_content/README.rst [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+.. contents::
+   :local:
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/server-tools/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/server-tools/issues/new?body=module:%20attachment_unindex_content%0Aversion:%2012.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (moylop260) [inline attachment_unindex_content/README.rst]: Change all URL to 11.0

File: attachment_unindex_content/README.rst [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    :alt: moylop260
+.. |maintainer-ebirbe| image:: https://github.com/ebirbe.png?size=40px
+    :target: https://github.com/ebirbe
+    :alt: ebirbe
+
+Current `maintainers <https://odoo-community.org/page/maintainer-role>`__:
+
+|maintainer-moylop260| |maintainer-ebirbe| 
+
+This module is part of the `OCA/server-tools <https://github.com/OCA/server-tools/tree/12.0/attachment_unindex_content>`_ project on GitHub.
```
Comment (moylop260) [inline attachment_unindex_content/README.rst]: same case here, please

File: attachment_unindex_content/README.rst [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fserver--tools-lightgray.png?logo=github
+    :target: https://github.com/OCA/server-tools/tree/12.0/attachment_unindex_content
```
Comment (moylop260) [inline attachment_unindex_content/README.rst]: Change 11.0

File: attachment_unindex_content/i18n/attachment_unindex_content.pot [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* attachment_unindex_content
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 12.0\n"
```
Comment (moylop260) [inline attachment_unindex_content/i18n/attachment_unindex_content.pot]: 11.0

Comment (moylop260) [reply attachment_unindex_content/i18n/attachment_unindex_content.pot]: Better, export it directly from odoo 11.0 in order to reduce the diff for the next change from translate bot

Comment (randall-vx) [reply attachment_unindex_content/i18n/attachment_unindex_content.pot]: Done !

File: attachment_unindex_content/README.rst (hunk 3 above)
Comment (randall-vx) [reply attachment_unindex_content/README.rst]: Done !

File: attachment_unindex_content/README.rst (hunk 2 above)
Comment (randall-vx) [reply attachment_unindex_content/README.rst]: Done !

File: attachment_unindex_content/README.rst (hunk 1 above)
Comment (randall-vx) [reply attachment_unindex_content/README.rst]: Done !

**Review by randall-vx — COMMENTED** [review]
Fixed version data to 11.0

**Comment by moylop260** [comment]
Merged manually since that it is a valid backport and the red status is unrelated

**Pull request merged**